src/Transport/PubSub/SubscriptionStore.cs

```diff
@@ -104,5 +104,7 @@ static IEnumerable<Type> GenerateMessageHierarchy(Type messageType)
         }
 
         readonly ConcurrentDictionary<Type, string[]> eventTypeToTopicListMap = new();
+
+        static readonly IEnumerable<string> SelectedSubscriptionColumns = new[] { nameof(SubscriptionEntity.Address) };
```

Comment:
```suggestion
        static readonly IEnumerable<string> AddressColumnFilter = new[] { nameof(SubscriptionEntity.Address) };
```
(Obviously would also need to make a corresponding change at usage site)

Reply:
Done. Good suggestion